### ISSUE TITLE
[EN][Dragon Soul][Hagara the Stormbinder]Hagara the Stormbinder encounter bugs

### ISSUE THREAD

**Issue by Aival94** [Atlantiss/BugTracker#9867]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

**Description**: 
Will describe all the current [Hagara the Stormbinder](http://www.wowhead.com/npc=55689/hagara-the-stormbinder) Normal bugs that i find and know.

**Current behaviour**: 
1. [Ice Lance](http://www.wowhead.com/spell=105316/ice-lance) can target the same target no matter if the target have already one.
2. [Lightning Storm](http://www.wowhead.com/spell=105465/lightning-storm) is hitting all raid members in the raid group no matter the difficulty and how many are alive.
3.(Minor) [Ice Tomb](http://www.wowhead.com/spell=104448/ice-tomb) objects can stay after a kill/wipe/reset of the boss.
4.(Minor) Sometimes portal to go outside Eye of Eternity stay after you pull boss and is also able to use it when you are in the middle of the encounter.

**Expected behaviour**: 
1. [Ice Lance](http://www.wowhead.com/spell=105316/ice-lance) should not be able to target same player as an other [Ice Lance](http://www.wowhead.com/spell=105316/ice-lance)(this can clearly be seen on many kills and also on solo kills where you see only one [Ice Lance](http://www.wowhead.com/spell=105316/ice-lance)).
2. [Lightning Storm](http://www.wowhead.com/spell=105465/lightning-storm) should hit 3/4 members in 10 man and 7/8 in 25 every 2-3 seconds and give them the debuff.
3.(Minor) [Ice Tomb](http://www.wowhead.com/spell=104448/ice-tomb) objects should disappear after the encounter end anyway.
4.(Minor) Portal to go outside Eye of Eternity should disappear after you start the encounter.

**Steps to reproduce the problem**:

1. Have a group of more than 8 members and do 10 or 25 man mode Hagara.
2. Pull Hagara.
3. See the results.

**Revision**: 5637

**Comment by stanq87** [comment]
a minor bug would be tha on   [Lightning Storm]  the  [Crystal Conductor]'s are in the  wrong places and that they lack the spawning animation. current https://youtu.be/uiIUJoq0vX8?t=2300  10n https://youtu.be/DnLnyjz25oc?t=135  10hc  https://youtu.be/SDHIRS-WcjY?t=170

**Comment by Aival94** [comment]
Updated with the Minor bug Stanq87 said. Will update with the fixes when are on live server as well except if someone will update it before me :)

**Comment by stanq87** [comment]
another one is that  if u die when inside the Ice Tomb and get resurected u still get massive damage from the debuf from the ice tomb,

**Comment by Aival94** [comment]
Implemeted Ice Tomb debuff bug reported by stanq87

**Comment by stanq87** [comment]
9. u can avoid the  [Ice Wave] by simply walking on the very edge of the platform https://youtu.be/-OXVCaT19oA?t=4776
10  [Focused Assault] has melee range but tanks who get out of it (5yards) still get dmg , http://www.wowhead.com/spell=107851/focused-assault http://cata-shoot.tauri.hu/?spell=107851 https://youtu.be/-OXVCaT19oA?t=4503 - u can seee here the warriors skills go red for out of range but the boss still hits

**Comment by Aival94** [comment]
Implemented ice waves bug and focused assault range as "need confirmation" as it might be just the packet timing between server and client. Both reported by stanq87.

**Comment by stanq87** [comment]
11.  just noticed that Focused Assault dmg ranges and the melle dmg range are not in sync, what i mean : focused assould should do 50% of weapon dmg bit o get these numbers:
a FA avarege 18902   avarege  26250
b FA avarege 21160   avarege 24943
c FA  avarege 23833  avarege 27181

but the FA avarege should be closer to 50% of melle avarege
OK nevermind it seems to be a warrior bug not a hagara bug
checked the stats on 2 difrent tanks and the numbers seems to be correct
melle a 32k fa avarege 15k
FA avarege 16k melle avarege 28k

poor warriors

**Comment by xmesaj2** [comment]
1,2,5 https://atlantiss.eu/changelog#rev5643

6 https://atlantiss.eu/changelog#rev5646

**Comment by Aival94** [comment]
@stanq87 so i will remove the 10 about focused assault range?

**Comment by stanq87** [comment]
@Aival94 something still seems to be wrong there, those  hits that land every 0,5sec need to have a range chek biuld inn before they apply dmg AND if the target is in fron of the boss which seems to not work properly

**Comment by stanq87** [comment]
#9836 

> Moving out of range cancelling Focused Assault seems to be a Normal/LFR-only mechanic.
> 
> See the following:
> https://www.youtube.com/watch?v=AgDQdlL7SSU
> Go to 0:30. You will see Hagara start channeling, and then the tank run out of range, cancelling Focused assault.
> 
> This is however not true for Heroic.
> See the following:
> https://www.youtube.com/watch?v=gsf80518Lgk
> The recording Resto druid has a focus frame which also shows the boss's cast bar.
> Go to 0:52. You will see their tank Heroic leap across the platform, far out of melee range.
> You will also see that doing so does not cancel Hagara's channel.
> 
> Focused Assault should not be parriable, dodgeable or blockable.
> See the following logs. Go into the "Damage by spell" tab and look at "Focused assault".
> http://www.worldoflogs.com/reports/pauko9t62zllmsjm/details/7/?s=1984&e=2098
> http://www.worldoflogs.com/reports/kq7wdq9sacp649yi/details/3/?s=1728&e=1962

**Comment by Aival94** [comment]
yeah stanq and i have moved away from focused assault and was not getting damage after sometime in normal. The point is if the range of the ability is more than 5 yards(melee range) or not.

**Comment by xmesaj2** [comment]
SpellRange: (Id 2) "Combat Range":
    MinRangeNegative = 0, MinRangePositive = 0
    MaxRangeNegative = 5, MaxRangePositive = 5

**Comment by stanq87** [comment]
another one is that when your traped inside a tomb your view is suposed to be  obscured by the ice,

**Comment by stanq87** [comment]
another one is a mage can use hes ice block in the ice tomb

**Comment by stanq87** [comment]
another one is  a druid a shamn or a druid can push back the tombs,  but when they do the tombs "walk" back to their  orginal position

**Comment by Aival94** [comment]
Removed all the fixes on changelog and implemented the bugs stanq said(removed all my comments to make it a little more clean).

**Comment by Aival94** [comment]
Updated

**Comment by xmesaj2** [comment]
> More than one Ice Lance can target a player.
> (Minor) Ice Tomb can be knocked back.

still valid?

**Comment by Aival94** [comment]
@xmesaj2 the ice lance yes. Removed the ice tomb

**Comment by Psychobird7** [comment]
Hagara can currently parry while stunned by Feedback. (on 25, if it makes a difference) 

This should obviously not happen.

**Comment by Flinvoltx** [comment]
Shattered ice shouldn't be interruptable, all guides mention 3 stacks swap and heavy damage of this spell, nothing about interrupt.
